net.hovancik.Stretchly.yaml

```diff
@@ -0,0 +1,57 @@
+app-id: net.hovancik.Stretchly
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
```

Comment:
22.08 maybe?

Reply:
There is no `org.freedesktop.Sdk.Extension.node14/x86_64/22.08` yet, otherwise I would update it to `22.08`.

Reply:
For some reason, when I attempt to update to node16, I can no longer build due to npm trying to install `electron-20.2.0.tgz`, which is not in the cached sources. Maybe a bug in the `flatpak-node-generator`?

Reply:
I figured this out. I needed to pass `--recursive` to `flatpak-node-generator` to pick up the dependencies in `app/package-lock.json`. That's why the version was missing. This allowed me to update to node16 and the 22.08 platform.